_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Bitcoin Core 0.13.0 Released!
+name: blog-release-0.13.0
+id: en-blog-release-0.13.0
+lang: en
+permalink: /en/2016/08/05/release-0.13.0/
+type: posts
+layout: post
+share: true
+version: 1
+---
+{% include _toc.html %}
+{% include _references.md %}
+
+We're pleased to announce the official release of Bitcoin Core 0.13.0.  During the six-month development cycle leading to this release, [dozens of contributors][release notes contributors] have made [hundreds of notable improvements][release notes changelog] to Bitcoin Core.  Among the many upgrades available in this release, the following may be especially interesting to miners, node operators, and wallet users:
```

Comment:
The links don't seem to work in the github rendered view (https://github.com/bitcoin-core/bitcoincore.org/blob/4d1897a01e4f136d2916d248c3c71d0d78479999/_posts/en/posts/2016-08-05-release-0-13-0.md), but I assume that is intentional.